src/taoensso/faraday.clj

```diff
@@ -369,15 +411,18 @@
 
   TableDescription
   (as-map [d]
-    {:name          (keyword (.getTableName d))
-     :creation-date (.getCreationDateTime d)
-     :item-count    (.getItemCount d)
-     :size          (.getTableSizeBytes d)
-     :throughput    (as-map (.getProvisionedThroughput  d))
-     :indexes       (as-map (.getLocalSecondaryIndexes  d)) ; DEPRECATED
-     :lsindexes     (as-map (.getLocalSecondaryIndexes  d))
-     :gsindexes     (as-map (.getGlobalSecondaryIndexes d))
-     :status        (utils/un-enum (.getTableStatus d))
+    {:name                 (keyword (.getTableName d))
+     :creation-date        (.getCreationDateTime d)
+     :item-count           (.getItemCount d)
+     :size                 (.getTableSizeBytes d)
+     :throughput           (as-map (.getProvisionedThroughput d))
+     :indexes              (as-map (.getLocalSecondaryIndexes d)) ; DEPRECATED
+     :lsindexes            (as-map (.getLocalSecondaryIndexes d))
+     :gsindexes            (as-map (.getGlobalSecondaryIndexes d))
+     :stream-specification (as-map (.getStreamSpecification d))
```

Comment:
My initial thinking was along the lines suggested by @dmarjenburgh, but I kept addition names short for consistency. I keep thinking we can do a mass rename eventually, for a breaking 2.0, but would agree that `:stream-spec` is closer to what DynamoDB has been using.